README.md

```diff
@@ -65,13 +64,19 @@ npm run install -- --ptb
 ```ps
 npm run install -- --canary
 ```
+### 3. Use. 
+To use latest changes
```

Comment:
I think "Use" should be renamed to "Compile" or something similar to match with the next instruction